docs/user-guide/src/bmo/external_inspection.md

```diff
@@ -0,0 +1,34 @@
+# External inspection
+
+Similar to the [status annotation](status_annotation.md), external inspection makes it possible to skip the inspection step.
+The difference is that the status annotation can only be used on the very first reconcile and allows setting all the fields under `status`.
+In contrast, external inspection limits the changes so that only HardwareDetails can be modified, and it can be used at any time when inspection is disabled or when there is no existing HardwareDetails data.
+
+External inspection is controlled through an annotation on the BareMetalHost.
+The annotation key is `inspect.metal3.io/hardwaredetails` and the value is a JSON representation of the BareMetalHosts `status.hardware` field.
+
+Here is an example with a BMH that has inspection disabled and is using the external inspection feature to add the HardwareDetails.
+
+```yaml
+apiVersion: metal3.io/v1alpha1
+kind: BareMetalHost
+metadata:
+  name: node-0
+  namespace: metal3
+  annotations:
+    inspect.metal3.io: disabled
```

Comment:
Probably this annotation can be highlighted in the description I could not get if it is needed with `inspect.metal3.io/hardwaredetails` annotation or not ? because you mentioned that external  annotation is only controlled through `inspect.metal3.io/hardwaredetails`

Reply:
Added a bit more about it above now. PTAL